src/js/utils/browser.js

```diff
@@ -169,10 +169,10 @@ export const IS_WINDOWS = (/Windows/i).test(USER_AGENT);
  * @const
  * @type {Boolean}
  */
-export const TOUCH_ENABLED = Dom.isReal() && (
+export const TOUCH_ENABLED = !!(Dom.isReal() && (
```

Comment:
Can you change this to `Boolean(...)`. I know its the same thing, but it's much easier to understand.